[2025-02-18T11:37:01.272+0000] {processor.py:186} INFO - Started process (PID=5055) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:37:01.272+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-18T11:37:01.274+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:37:01.274+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:37:01.277+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:37:01.277+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-18T11:37:01.278+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:37:01.277+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-18T11:37:01.278+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:37:01.287+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2025-02-18T11:38:00.898+0000] {processor.py:186} INFO - Started process (PID=5306) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:38:00.902+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-18T11:38:00.904+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:38:00.904+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:38:00.907+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:38:00.907+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-18T11:38:00.907+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:38:00.907+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-18T11:38:00.908+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:38:00.918+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2025-02-18T11:40:04.039+0000] {processor.py:186} INFO - Started process (PID=5800) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:40:04.042+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-18T11:40:04.046+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:40:04.045+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:40:04.052+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:40:04.051+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-18T11:40:04.052+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:40:04.052+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-18T11:40:04.053+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:40:04.069+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2025-02-18T11:41:05.760+0000] {processor.py:186} INFO - Started process (PID=6108) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:41:05.762+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-18T11:41:05.765+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:41:05.764+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:41:05.769+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:41:05.769+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-18T11:41:05.770+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:41:05.770+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-18T11:41:05.770+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:41:05.782+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2025-02-18T11:42:05.841+0000] {processor.py:186} INFO - Started process (PID=6345) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:42:05.844+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-18T11:42:05.848+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:42:05.848+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:42:05.854+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:42:05.853+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-18T11:42:05.855+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:42:05.854+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-18T11:42:05.855+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:42:05.870+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2025-02-18T11:43:10.919+0000] {processor.py:186} INFO - Started process (PID=6763) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:43:10.922+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-18T11:43:10.926+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:43:10.926+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:43:10.932+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:43:10.931+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-18T11:43:10.932+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:43:10.932+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-18T11:43:10.933+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:43:10.949+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2025-02-18T11:44:15.952+0000] {processor.py:186} INFO - Started process (PID=7178) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:44:15.955+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-18T11:44:15.959+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:44:15.959+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:44:15.964+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:44:15.963+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-18T11:44:15.964+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:44:15.964+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-18T11:44:15.964+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:44:15.977+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2025-02-18T11:45:20.173+0000] {processor.py:186} INFO - Started process (PID=7539) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:45:20.175+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-18T11:45:20.177+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:45:20.177+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:45:20.182+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:45:20.181+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-18T11:45:20.182+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:45:20.182+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-18T11:45:20.182+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:45:20.193+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2025-02-18T11:46:19.485+0000] {processor.py:186} INFO - Started process (PID=7840) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:46:19.489+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-18T11:46:19.492+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:46:19.492+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:46:19.497+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:46:19.497+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-18T11:46:19.497+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:46:19.497+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-18T11:46:19.498+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:46:19.508+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2025-02-18T11:47:20.046+0000] {processor.py:186} INFO - Started process (PID=8082) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:47:20.049+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-18T11:47:20.052+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:47:20.052+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:47:20.056+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:47:20.055+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-18T11:47:20.056+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:47:20.056+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-18T11:47:20.057+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:47:20.067+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.034 seconds
[2025-02-18T11:48:22.174+0000] {processor.py:186} INFO - Started process (PID=8323) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:48:22.176+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-18T11:48:22.178+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:48:22.178+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:48:22.182+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:48:22.182+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-18T11:48:22.182+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:48:22.182+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-18T11:48:22.182+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:48:22.191+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2025-02-18T11:58:23.921+0000] {processor.py:186} INFO - Started process (PID=8617) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:58:23.925+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-18T11:58:23.927+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:58:23.927+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:58:23.930+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:58:23.930+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-18T11:58:23.931+0000] {logging_mixin.py:190} INFO - [2025-02-18T11:58:23.931+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-18T11:58:23.931+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-18T11:58:23.943+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
